docs/installation/mac.md

```diff
@@ -274,85 +326,87 @@ The next exercise demonstrates how to do this.
+| info           | inspect          | Displays a machine's details.                            |
+| ip             | ip               | Displays the machine's ip address.                       |
+| shellinit      | env              | Displays shell commands needed to configure your shell to interact with a machine |
+| delete         | rm               | Removes a machine.                                       |
+| download       | -                | Not applicable.                                          |
+| upgrade        | upgrade          | Upgrades a machine's Docker client to the latest stable release. |
 
 
-## Uninstallation 
+## Upgrade Docker Toolbox
 
-1. Go to the [boot2docker/osx-installer ](
-   https://github.com/boot2docker/osx-installer/releases/latest) release page. 
+To upgrade Docker Toolbox, download an re-run [the Docker Toolbox
+installer](https://github.com/docker/toolbox/releases/tag/v1.7.0).
 
-2. Download the source code by clicking `Source code (zip)` or
-   `Source code (tar.gz)` in the "Downloads" section.
 
-3. Extract the source code.
+## Uninstall Docker Toolbox 
 
-4. Open a terminal on your local machine.
+To uninstall, do the following:
 
-5. Change to the directory where you extracted the source code:
+1. List your machines.
 
-        $ cd <path to extracted source code>
+      $ docker-machine ls
+      NAME                ACTIVE   DRIVER       STATE     URL                         SWARM
+      dev                 *        virtualbox   Running   tcp://192.168.99.100:2376   
+      my-docker-machine            virtualbox   Stopped                               
+      my-docker-vm                 virtualbox   Stopped  
 
-6. Make sure the uninstall.sh script is executable:
+2. Remove each machine.
 
-        $ chmod +x uninstall.sh
+      $ docker-machine rm dev
+      Successfully removed dev
+    
+   Removing a machine deletes its VM from virtual box and from the
+   `~/.docker/machine/machines` directory.
 
-7. Run the uninstall.sh script:
+3. Remove the Docker CLI and Kitematic from your "Applications" folder.
 
-        $ ./uninstall.sh
+4. Remove the `docker`, `docker-compose`, and `docker-machine` commands from the `/usr/local/bin` folder.
 
+      $ rm /usr/local/bin/docker
+      
+5. Delete the `~/.docker` folder from your system.
 
-## Learning more and acknowledgement
 
-Use `boot2docker help` to list the full command line reference. For more
-information about using SSH or SCP to access the Boot2Docker VM, see the README
-at  [Boot2Docker repository](https://github.com/boot2docker/boot2docker).
+## Learning more 
 
-Thanks to Chris Jones whose [blog](http://viget.com/extend/how-to-use-docker-on-os-x-the-missing-guide)  
-inspired me to redo this page.
+Use `docker-machine help` to list the full command line reference for Docker Machine. For more
+information about using SSH or SCP to access a VM, see [the Docker Machine
+documentation](http://docs.docker.com/machine/).
```

Comment:
Should there be a mention of Kitematic here? It's not described at all, and can actually be a nice way to get started with Docker (also to access the `docker cli`